packages/plugin/webpack/test/WebpackConfig_spec.ts

```diff
@@ -308,113 +263,6 @@ describe('WebpackConfigGenerator', () => {
     });
   });
 
-  describe('getPreloadConfigForEntryPoint', () => {
```

Comment:
Same here, we have new functions for generating configs, we should test those new functions if we're deleting the old tests